api/test/unit/views/media_views_test.py

```diff
@@ -68,6 +68,32 @@ def send(session, req, **kwargs):
     return fixture
 
 
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "media_type, media_factory",
+    [
+        ("images", ImageFactory),
+        ("audio", AudioFactory),
+    ],
+)
+def test_list_query_count(api_client, media_type, media_factory):
+    media = media_factory.create()
+
+    results = ([MagicMock(identifier=str(media.identifier))], 1, 1)
```

Comment:
The mock here makes sense and the new test is good, but having a single result makes the 1 query count suspicious :slightly_smiling_face: A full page of results would make the test more convincing long term as changes are made to the serializer and rest of the view.

Reply:
Updated the test to return 20 results in 48c463f.